### ISSUE TITLE
Change status_quo from using values module to taxonomy reference

### ISSUE THREAD

**Issue by heliogabal** [saurabhd/hk_realestate#58]
Right now, the field_status_quo in the apartment content type used to utilize the values module to create the defined values for the list field. As all other similar fields (category, apartment_type, borough etc.) are using taxonomies, we should harmonize that and switch the status quo to using taxonomy as well. 

So we need to create another field based on taxonomy terms, update the values from the old field, delete the old and show the new on all occurencies of the field. The list of status quo entries that should be replicated in the taxonomy:

first_occupancy|First Occupancy
first_occupancy_after_renovation|First Occupancy after Renovation
well_tended|Well-Tended
very_well_tended|Very well tended
normal|Normal
modernised|Modernised
completely_refurbished|Completely Refurbished
partially_restored|Partially restored
renovated|Renovated
in_need_of_renovation|In need of renovation
in_need_of_modernization|In need of modernization
not_developed_yet|Not developed yet
no_information|No Information

Once the switch is done, we can also deactivate and uninstall the values module, as it is no longer needed.

**Comment by heliogabal** [comment]
you should also change the order in the taxonomy edit form and put the description under the name.

Then, somehow the translations got mixed up, all the terms are saying the are German but it's the English Term. Don't know it that's in a feature, but if it is please adjust the terms and their translations:
 Completely Refurbished
  First Occupancy Erstbezug
  First Occupancy after Renovation Erstbezug nach Renovierung
  In need of modernization Modernisierungsbefürftig
  In need of renovation Sanierungsbedürftig
  Modernised Modernisiert
  N/A Keine Angabe
  No Information Keine Information
  Non converted Nicht ausgebaut
  Normal Normal
  Partially restored Teilsaniert
  Renovated Renoviert
  Very well-tended Sehr gepflegt
  Well-tended Gut gepflegt

**Comment by heliogabal** [comment]
I disbled and uninstalled the values module, thanks to this issue.
